### PR TITLE
feat(autoencoder): add batch normalization layer

### PR DESCRIPTION
**Pull request by HiroIshida**

### CODE REVIEW

**Comment by HiroIshida** [comment]
Found that there is no improvement by batch normalization in autoencoder. 
Also, using bath normalization in VAE is known to be a bad idea. https://stats.stackexchange.com/questions/420543/distorted-validation-loss-when-using-batch-normalization-in-convolutional-autoen
